ssl/s3_enc.c

```diff
@@ -231,7 +231,7 @@ int ssl3_change_cipher_state(SSL *s, int which)
             goto err;
         else
             /*
-             * make sure it's intialized in case we exit later with an error
+             * make sure it's initialised in case we exit later with an error
```

Comment:
To clarify: British English or Oxford English? https://en.wikipedia.org/wiki/Oxford_spelling